gsoc2018-bharat/README.md

```diff
@@ -2,16 +2,27 @@
 
 This project aims to generate complex word embeddings for Out-Of-Vocabulary entities. After completion, the package will be able to generate pre-trained embeddings, improve them by generating embeddings on-the-fly, evaluate the benchmarks using pre-trained embeddings, and make the same evaluations on the imporoved embeddings.
 
-## Requirements
+### Getting started
 
-All the libraries used under this project are included in the file, requirements.txt. To install, just run the command
+Here is how you can start training and testing the model yourself. First, clone the repository on your local machine. I assume you are familiar with Git, and have it installed on your system prior to using this repo.
 
 ```shell
+$ git clone git@github.com:tramplingWillow/embeddings.git
```

Comment:
for example, this would be `dbpedia/embeddings.git` since we are merging to dbpedia, right ?

Reply:
Yes.